django/contrib/gis/gdal/srs.py

```diff
@@ -53,6 +57,8 @@ def __init__(self, srs_input='', srs_type='user'):
         if srs_type == 'wkt':
             self.ptr = capi.new_srs(c_char_p(b''))
             self.import_wkt(srs_input)
+            if axis_order == 'traditional' and GDAL_VERSION >= (3, 0):
+                capi.set_axis_strategy(self.ptr, OAMS_TRADITIONAL_GIS_ORDER)
```

Comment:
Might make sense to raise an exception if `axis_order != 'traditional' and GDAL_VERSION < (3, 0)`.